src/org/usfirst/frc/team166/robot/Robot.java

```diff
@@ -60,29 +63,36 @@ public void robotInit() {
 		vision = new Vision();
 		intakeRoller = new IntakeRoller();
 		aManipulators = new AManipulators();
+		shooterLock = new ShooterLock();
 
 		// autochooser
 		autoChooser = new SendableChooser();
 
-		// triggers
-		copilotRightTrigger = new CopilotRightTrigger();
-		copilotLeftTrigger = new CopilotLeftTrigger();
-
 		oi = new OI();
-		// instantiate the command used for the autonomous period
-		lowGearCommand = new LowGear();
 
 		// auto chooser commands
 		autoChooser.addDefault("FarLeftAuto", new FarLeftAuto());
-		autoChooser.addObject("MidLeftAuto", new MidLeftAuto());
-		autoChooser.addObject("MidAuto", new MidAuto());
-		autoChooser.addObject("MidRightAuto", new MidRightAuto());
-		autoChooser.addObject("FarRightAuto", new FarRightAuto());
+		autoChooser.addObject("Paper Weight", null);
+		autoChooser.addObject("All Up Mid Auto", new AllUpMidAuto());
+		autoChooser.addObject("All Down Mid Auto", new AllDownMidAuto());
+		autoChooser.addObject("All Down Position 2 Auto", new AllDownPositionTwoAuto());
+		autoChooser.addObject("All Up Position 2 Auto", new AllUpPositionTwoAuto());
+
+		// autoChooser.addObject("MidLeftAuto", new MidLeftAuto());
+		// autoChooser.addObject("MidRightAuto", new MidRightAuto());
+		// autoChooser.addObject("FarRightAuto", new FarRightAuto());
+		// autoChooser.addObject("Uber Auto", new UberAuto());
+
+		// Hard code for auto
+		// autonomousCommand = new FarLeftAuto();
+		// autonomousCommand = new AllDownAuto();
+		// autonomousCommand = new AllUpAuto();
+		// autonomousCommand = new PaperWeightAuto();
```

Comment:
We have git. We can magically step back in time to look at or use ANY commit. There is no reason to keep commented-out code around.